src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -495,20 +495,27 @@ void RasterizerVulkan::Query(GPUVAddr gpu_addr, VideoCore::QueryType type,
 
 void RasterizerVulkan::FlushAll() {}
 
-void RasterizerVulkan::FlushRegion(CacheAddr addr, u64 size) {
+void RasterizerVulkan::FlushRegion(VAddr addr, u64 size) {
+    if (!addr || !size) {
```

Comment:
```suggestion
    if (addr == 0 || size == 0) {
```

Since these are integral values, not bools.